src/models/pmv.js

```diff
@@ -0,0 +1,216 @@
+ *    the applicability limits of the model.
+ *    The ASHRAE 55 2020 limits are 10 < tdb [°C] < 40, 10 < tr [°C] < 40,
+ *    0 < vr [m/s] < 2, 1 < met [met] < 4, and 0 < clo [clo] < 1.5.
+ *    The ISO 7730 2005 limits are 10 < tdb [°C] < 30, 10 < tr [°C] < 40,
+ *    0 < vr [m/s] < 1, 0.8 < met [met] < 4, 0 < clo [clo] < 2, and -2 < PMV < 2.
+ * @property {boolean} airspeed_control - This only applies if standard = "ASHRAE".
+ *    Default is True. By default, it is assumed that the occupant has control over the airspeed.
+ *    In this case, the ASHRAE 55 Standard does not impose any airspeed limits.
+ *    On the other hand, if the occupant has no control over the airspeed,
+ *    the ASHRAE 55 imposes an upper limit for v which varies as a function of
+ *    the operative temperature, for more information please consult the Standard.
+ * @public
+ */
+
+/**
+ * Returns Predicted Mean Vote (`PMV`_) calculated in accordance with main thermal comfort Standards.
+ *
+ * The PMV is an index that predicts the mean value of the thermal sensation votes (self-reported perceptions)
+ * of a large group of people on a sensation scale expressed from –3 to +3 corresponding to the categories: cold, cool,
+ * slightly cool, neutral, slightly warm, warm, and hot. [1]_
+ *
+ * While the PMV equation is the same for both the ISO and ASHRAE standards, in the ASHRAE 55 PMV equation, the SET is
+ * used to calculate the cooling effect first, this is then subtracted from both the air and mean radiant temperatures,
+ * and the differences are used as input to the PMV model, while the airspeed is set to 0.1m/s. Please read more in the
+ * Note below.
+ *
+ * [1]    ANSI, & ASHRAE. (2020). Thermal Environmental Conditions for Human Occupancy. Atlanta.
+ *
+ * [2]    ISO. (2005). ISO 7730 - Ergonomics of the thermal environment — Analytical determination and interpretation
+ *      of thermal comfort using calculation of the PMV and PPD indices and local thermal comfort criteria.
+ *
+ * This is a version that supports scalar arguments.
+ * @see {@link pmv_array} for a version that supports arrays.
+ *
+ * @public
+ * @memberof models
+ * @docname Predicted Mean Vote (PMV)
+ *
+ * @param {number} tdb - dry bulb air temperature, default in [°C] in [°F] if `units` = 'IP'
+ * @param {number} tr - mean radiant temperature, default in [°C] in [°F] if `units` = 'IP'
+ * @param {number} vr - relative air speed, default in [m/s] in [fps] if `units` = 'IP'
+ * @param {number} rh - relative humidity, [%]
+ * @param {number} met - metabolic rate
+ * @param {number} clo - clothing insulation
+ * @param {number} wme - external work, wme default 0
+ * @param {"ISO"|"ASHRAE"|"iso"|"ashrae"} standard - comfort standard used for calculation
+ * @param {pmvKwargs} kwargs - additional arguments
+ *
+ * @returns {number} pmv - Predicted Mean Vote
+ *
+ * @notes You can use this function to calculate the `PMV`_ [1]_ [2]_.
+ * _PMV:
```

Comment:
is this meant to show above the links? if that is the case you might want to remove the `@see` tags, otherwise they will get rendered under a Related subtitle